app/importer.js

```diff
@@ -234,9 +234,17 @@ importer.on('add-cookies', (e, cookies) => {
       httpOnly: cookies[i].httponly,
       expirationDate: cookies[i].expiry_date
     }
+    // Bypassing cookie mistach error in
```

Comment:
typo here- should be `mismatch`

So this is avoiding the error by not importing Google cookies? Is that an acceptable solution?

Reply:
They are like blackbox even if I can see the name and value. And it is not a single cookie cause problem so I narrowed them down to a small set.